### PR TITLE
Netmhcpan41

### PR DESCRIPTION
**Pull request by amitschang**
netmhcpan4.1 requires special treatment. See openvax#152, and the PR referenced therein. This is basically that PR plus some extra code, which seems to work.

I don't see a lot of movement on that repo, hence the fork here

### CODE REVIEW

**Comment by vkluzner** [comment]
@amitschang  - Was that forked from original mhctools repo? If yes, what was the reason for it?

**Review by vkluzner — APPROVED** [review]

**Comment by amitschang** [comment]
@vkluzner, it is because netmhcpan4.1 requires special treatment. See https://github.com/openvax/mhctools/issues/152, and the PR referenced therein. This is basically that PR plus some extra code, which seems to work. 

I don't see a lot of movement on that repo, hence the fork here

File: mhctools/netmhc_pan4.py [hunk 1]
```diff
@@ -55,6 +55,43 @@ def __init__(
             extra_flags=flags + extra_flags,
             process_limit=process_limit)
 
+class NetMHCpan41(BaseCommandlinePredictor):
```
Comment (jamienoss) [inline mhctools/netmhc_pan4.py:58]: Was this copied from some existing code, an older version, or something?

Comment (jamienoss) [reply mhctools/netmhc_pan4.py:58]: Ah, I see, the code above.

File: mhctools/parsing.py [hunk 2]
```diff
@@ -400,6 +400,44 @@ def parse_netmhcpan4_stdout(
         rank_index=12 if mode == "elution_score" else 13,
         transforms=transforms)
 
+def parse_netmhcpan41_stdout(
```
Comment (jamienoss) [inline mhctools/parsing.py:403]: ~Again, I see, this is just a carbon copy of the code above - well almost a cc. Some values are diff.~

It's from https://github.com/openvax/mhctools/blob/master/mhctools/parsing.py#L441-L477

**Comment by jamienoss** [comment]
This seems to be resolved upstream by https://github.com/openvax/mhctools/pull/163.

@amitschang can you confirm, please, cheers?

FYI, I'm gonna close this for now, please reopen if need be.

**Pull request closed without merging**